manifests/g/GLab/GLab/1.27.0/GLab.GLab.locale.en-US.yaml

```
@@ -0,0 +1,34 @@
+# Created using wingetcreate 1.2.5.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.4.0.schema.json
+
+PackageIdentifier: GLab.GLab
+PackageVersion: 1.27.0
+PackageLocale: en-US
+Publisher: GitLab
+PublisherUrl: https://gitlab.com/gitlab-org/cli
+PublisherSupportUrl: https://gitlab.com/gitlab-org/cli/-/issues
+Author: GitLab
+PackageName: GitLab CLI

```

```suggestion
PackageName: glab
```
Must match Control Panel / Apps&Features